Client/Coordinators/EnhancedTrackingProtectionCoordinator.swift

```diff
@@ -43,7 +43,15 @@ class EnhancedTrackingProtectionCoordinator: BaseCoordinator,
     }
 
     func start() {
-        router.push(enhancedTrackingProtectionMenuVC)
+        enhancedTrackingProtectionMenuVC.modalPresentationStyle = .custom
+        enhancedTrackingProtectionMenuVC.transitioningDelegate = self
+        let keyWindow = UIApplication.shared.windows.filter {$0.isKeyWindow}
+        if var topController = keyWindow.first?.rootViewController {
+            while let presentedViewController = topController.presentedViewController {
+                topController = presentedViewController
+            }
+            topController.present(enhancedTrackingProtectionMenuVC, animated: true, completion: nil)
+        }
```

Comment:
Why was this needed to replace `router.push(enhancedTrackingProtectionMenuVC)` ? We should keep pushing from the router with the coordinator

Reply:
@lmarceau  The issue is that the enhancedTrackingProtectionMenuVC is pushed with an empty controller if it is pushed from the last navigation controller, not the top controller.

Reply:
Ah, I think it's because there's no `rootViewController` to this navigation controller. Can you try `router.setRootViewController(enhancedTrackingProtectionMenuVC)`? (like it was done for settings coordinator [here](https://github.com/mozilla-mobile/firefox-ios/blob/aa74c92e634a34e15db231bfee14df4a4d9ddff0/Client/Coordinators/SettingsCoordinator.swift#L42))

Reply:
@lmarceau  Yes, I tried this, but it keeps crashing